testsuite/MDAnalysisTests/test_streamio.py

```diff
@@ -19,7 +19,7 @@
 import MDAnalysis
 import MDAnalysis.lib.util as util
 import MDAnalysis.tests.datafiles as datafiles
-from MDAnalysisTests.test_coordinates import RefAdKSmall
+from MDAnalysisTests.coordinates.reference import RefAdKSmall
```

Comment:
Ok I think I found it. The error was in `setup.py`. It didn't install the new coordinates modules.